.nvmrc

```diff
@@ -1 +1 @@
-v12.19.0
\ No newline at end of file
+v15.0.0
```

Comment:
Can this be 15.x.x or just 15?
I think v15.0.1 is the stable at the moment

Reply:
If you change it to `v15/*` that will allow any v15 version

Reply:
Updated nvmrc and engine in package.json